lib/widgets/content.dart

```diff
@@ -99,10 +99,9 @@ class Paragraph extends StatelessWidget {
   final ParagraphNode node;
 
   static TextStyle getTextStyle(BuildContext context) => const TextStyle(
-    fontFamily: 'Source Sans 3',
-    fontSize: 14,
-    height: (17 / 14),
-  ).merge(weightVariableTextStyle(context));
+    fontSize: kBaseFontSize,
+    height: (17 / kBaseFontSize),
+  );
```

Comment:
For a further nice cleanup (probably in a further cleanup commit at the end):
```dart
  static const textStyle = TextStyle(
    fontSize: kBaseFontSize,
    height: (17 / kBaseFontSize));
```

Since we're no longer looking at `context`, this can become a constant.

Reply:
Ah indeed; I meant to do this as a followup here but forgot.